.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: false
+language: python
+python:
+- '2.7'
+- '3.2'
+- '3.3'
+- '3.4'
+- '3.5'
+install: pip install -r requirements.txt
+script:
+- ./scripts/travis_test.sh
```

Comment:
I'm quite a fan of using `tox`, and then making travis-ci call `tox`. If you want to do this, checkout https://github.com/yola/demands as an example